### PR TITLE
debug error page in development was receiving reset context

### PR DESCRIPTION
**Pull request by bdtomlin**
## Purpose
Fix debug error page

## Description
This fixes the following issue: https://github.com/luckyframework/lucky/issues/933

## Checklist
* [x] - An issue already exists detailing the issue/or feature request that this PR fixes
* [x] - All specs are formatted with `crystal tool format spec src`
* [x] - Inline documentation has been added and/or updated
* [x] - Lucky builds on docker with `./script/setup`
* [x] - All builds and specs pass on docker with `./script/test`

### CODE REVIEW

File: spec/lucky/error_handling_spec.cr [hunk 1]
```diff
@@ -121,14 +123,16 @@ end
 private def handle_error(
   format : Symbol = :html,
   show_debug_output : Bool = false,
-  error : Exception = UnhandledError.new
+  error : Exception = UnhandledError.new,
+  status_code : Int32 = 200
```
Comment (paulcsmith) [inline spec/lucky/error_handling_spec.cr]: I believe that the default is not used in any of the specs so can be removed. But I may have overlooked something so if it’s used it is of course ok to leave as is :)

Comment (bdtomlin) [reply spec/lucky/error_handling_spec.cr]: Yeah, the default is still used in a couple examples:
https://github.com/luckyframework/lucky/blob/4fd49c186fba7f5d3f08d8acea5685a725f6dc9b/spec/lucky/error_handling_spec.cr#L36-L55

Comment (paulcsmith) [reply spec/lucky/error_handling_spec.cr]: Perfect. I’ll go ahead and merge it

**Comment by paulcsmith** [comment]
Thanks 😃

**Comment by bdtomlin** [comment]
No problem. Glad to help, even if just a little 😀

**Pull request merged**